code/game/objects/items/kitchen.dm

```diff
@@ -102,6 +102,45 @@
 	. = ..()
 	AddElement(/datum/element/easily_fragmented, PLASTIC_BREAK_PROBABILITY)
 
+/obj/item/knife/kitchen/silicon
+	name= "Kitchen Toolset"
+	icon_state = "SiliKnife"	
```

Comment:
nit, but `icon_state` stuff should really be `snake_case`, so this would be `sili_knife` instead (it's developer facing only, so it's fine your flavor isn't lost). just change the name in the DMI and change it here as well